tests/std/tests/P0896R4_ranges_alg_move/test.cpp

```diff
@@ -9,75 +9,69 @@
 
 #include <range_algorithm_support.hpp>
 
+using namespace std;
+
 struct int_wrapper {
     int val                 = 10;
     constexpr int_wrapper() = default;
     constexpr int_wrapper(int x) : val{x} {}
+    int_wrapper(const int_wrapper&) = default;

```

Comment:
@CaseyCarter 🕵️

Reply:
Hrm, I must've reverted it while trying to hack around the `/analyze` issue. Fixed.

Reply:
And here I am removing the code just to realize that ninja Casey was faster than me